docs/source/03_HED_formats.md

```diff
@@ -1263,7 +1263,7 @@ The validator must also check to make sure that `Onset` and `Offset` tags are
 properly matched within the data recording.
 In particular every `Offset` tag group must correspond to a preceding `Onset` tag group.
 
-See [**ONSET_OFFSET_INSET_ERROR**](./Appendix_B.md#onset_offset_inset_error) for details on the
+See [**TEMPORAL_TAG_ERROR**](./Appendix_B.md#temporaltagerror) for details on the

```

Comment:
See [**TEMPORAL_TAG_ERROR**](./Appendix_B.md#temporal_tag_error) for details on the